src/Checkbox/Checkbox.jsx

```diff
@@ -49,10 +50,13 @@ export const Checkbox = ({
         name={name}
         checked={isChecked}
         style={{ display: 'none' }}
-        onChange={handleChange}
+        onChange={disabled ? () => {} : handleChange}
```

Comment:
I think we should better check the disabled state inside the `handleChange` function

Reply:
29d79de4d68c55cac0aa453273d77d401635756b